carbon-p2-plugin/pom.xml

```diff
@@ -34,91 +34,114 @@
     <name>Maven P2 Profile Generation Plugin</name>
 
     <dependencies>
-        	<dependency>
-			<groupId>org.apache.maven</groupId>
-			<artifactId>maven-plugin-api</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.maven</groupId>
-			<artifactId>maven-core</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.tycho</groupId>
-			<artifactId>tycho-equinox</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.tycho</groupId>
-			<artifactId>tycho-equinox-launching</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.tycho</groupId>
-			<artifactId>tycho-p2-facade</artifactId>
-		</dependency>
-        	<dependency>
-            		<groupId>org.apache.maven</groupId>
-            		<artifactId>maven-project</artifactId>
-        	</dependency>
-        	<dependency>
-            		<groupId>junit</groupId>
-            		<artifactId>junit</artifactId>
-            		<scope>test</scope>
-        	</dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-plugin-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>tycho-equinox</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>tycho-equinox-launching</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>tycho-p2-facade</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-project</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugin-tools</groupId>
+            <artifactId>maven-plugin-annotations</artifactId>
+            <version>3.3</version>

```

Comment:
This is fixed in https://github.com/wso2/carbon-maven-plugins/pull/8/files#diff-2da93a3b99922c8b8ee654b88aef0bbaR69